packages/lit-dev-content/site/docs/v3/tools/testing.md

```diff
@@ -75,3 +75,48 @@ export default {
 };
 ```
 
+## Using WebdriverIO
+
+[WebdriverIO](https://webdriver.io) is a good alternative for your component or end-to-end tests. It has very compelling advantages like support for [mocking](https://webdriver.io/docs/component-testing/mocking) and [code coverage](https://webdriver.io/docs/component-testing/coverage) reporting.
```

Comment:
Minor nit:

```suggestion
[WebdriverIO](https://webdriver.io) is a good option for your component or end-to-end tests. It has very compelling advantages like support for [mocking](https://webdriver.io/docs/component-testing/mocking) and [code coverage](https://webdriver.io/docs/component-testing/coverage) reporting.
```

Instead of `alternative`, we can maybe say `option`. Thus focussing more on options vs preferences.